app/assets/javascripts/app/account_transaction_report.coffee

```diff
@@ -0,0 +1,26 @@
+class AccountTransactionReport
+  constructor: (@$element) ->
+    return unless @$element.length > 0
+    @registerExportAllHandler()
+
+  registerExportAllHandler: ->
+    $('#js--exportAll').click (event) => @exportAllClicked(event)
```

Comment:
Whoops. Yea, initially I meant to do something like `$('#table_billing').find('#js--exportAll')`, but that probably isn't even worth it since the `js--exportAll` is unique.